doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -286,6 +286,7 @@ nRF9160 samples
 
     * CoAP max message size is set to 1280 by default.
     * Number of SenML CBOR records is set to a higher value to cope with data exchange after registration with Coiote server.
+    * Sample has been switched to use TF-M instead of SPM.
```

Comment:
This would be better to add under a `* Changed:` block

Reply:
Done.